dso/labels.py

```diff
@@ -91,6 +91,18 @@ class CveCategorisationLabel(Label):
     value: dso.cvss.CveCategorisation
 
 
+@dataclasses.dataclass(frozen=True)
+class ExtraComponentReference:
+    component_reference: cm.ComponentIdentity
+    purpose: tuple[str] | None
+
+
+@dataclasses.dataclass(frozen=True)
+class ExtraComponentReferencesLabel(Label):
+    name = 'ocm.software/ocm-gear/extra-component-references'
```

Comment:
Shouldn't this rather be a general "defined / well-known" ocm label, and not an `ocm-gear` / `delivery-gear` specific?